aait/backend/group-2A/Application/Features/User/Handlers/Commands/CreateUserCommand.cs

```diff
@@ -1,38 +1,52 @@
 ﻿using System.Threading;
 using System.Threading.Tasks;
+using Application.Contracts.Identity;
 using Application.Contracts.Persistance;
 using Application.DTO.Post.Validation;
 using Application.DTO.UserDTO.Validator;
 using Application.Exceptions;
 using Application.Features.User.Request.Commands;
+using Application.Model;
 using AutoMapper;
 using Domain.Entities;
 using MediatR;
 
 namespace Application.Features.UserFeatures.Handlers.Commands
 {
-    public class CreateUserCommandHandler : IRequestHandler<CreateUserCommand, int>
+    public class CreateUserCommandHandler : IRequestHandler<CreateUserCommand, AuthResponse>
     {
         private readonly IUnitOfWork _unitOfWork;
         private readonly IMapper _mapper;
+        private readonly IAuthService _authService;
 
-        public CreateUserCommandHandler(IUnitOfWork unitOfWork, IMapper mapper)
+
+        public CreateUserCommandHandler(IUnitOfWork unitOfWork, IMapper mapper, IAuthService authService)
         {
-            _mapper = mapper;
+            _mapper = mapper; 
             _unitOfWork = unitOfWork;
+            _authService = authService;
         }
 
-        public async Task<int> Handle(CreateUserCommand request, CancellationToken cancellationToken){
+        public async Task<AuthResponse> Handle(CreateUserCommand request, CancellationToken cancellationToken){
             var validator = new CreateUserDTOValidator(_unitOfWork.userRepository);
             var validationResult = await validator.ValidateAsync(request.CreateUser);
             if (!validationResult.IsValid)
             {
+                
                 throw new ValidationException(validationResult);
             }
             var user = _mapper.Map<Domain.Entities.User>(request.CreateUser);
             await _unitOfWork.userRepository.Add(user);
+            Console.WriteLine("reached Here");
```

Comment:
remove : Console.WriteLine("reached Here"); don't push print codes to remote repository. remove them once you're ready to push them to remote repo.